docs/versioned_docs/version-v2.6.0/02-Guidance/Service-to-service Correlation/use-service-to-service-correlation-in-service-bus.md

```diff
@@ -0,0 +1,351 @@
+
+The following diagram shows how an Web API calls a Service Bus with correlation tracking:
+![Messaging correlation diagram](/media/service-to-service-api-worker-diagram-example.png)
+
+## How does this work?
+Three kind of correlation ID's are used to create the relationship:
+* **Transaction ID**: this ID is the one constant in the diagram. This ID is used to describe the entire transaction, from beginning to end. All telemetry will be tracked under this ID.
+* **Operation ID**: this ID describes a single operation within the transaction. This ID is used within a service to link all telemetry of that service together.
+* **Operation Parent ID**: this ID is create the parent/child link across services. When service A calls service B, then service A is the so called 'parent' of service B.
+
+The following list shows each step in the diagram:
+1. The initial call in this example doesn't contain any correlation headers. This can be seen as a first interaction call to a service. 
+2. Upon receiving at service A, the application will generate new correlation information. This correlation info will be used when telemetry is tracked on the service.
+3. When a call is made to service B, the **transaction ID** is sent but also the **operation parent ID** in the form of a hierarchical structure.
+4. The `jkl` part of this ID, describes the new parent ID for service B (when service B calls service C, then it will use `jkl` as parent ID)
+5. The user receives both the **transaction ID** and **operation parent ID** in their final response.
+
+> 💡 Additional configuration is available to tweak this functionality, see the [dedicated Arcus Messaging feature documentation](https://messaging.arcus-azure.net/Features/message-pumps/service-bus) for more in-depth information on Azure Service Bus correlation.
+
+## Service Bus demonstration
+In this user-guide, a fictive API and Service Bus application will be used to represent Service A and Service B of the diagram. Both services will be adapted to use service-to-service correlation.
+
+* **Order API** (Service A): receives an request to order a product, sends the order to process.
+* **Order Worker** (Service B): receives the order request and processes the order.
+
+The user interacts with the **Product API** to order their product. Internally, the **Order API** will contact the **Order Worker** to further process the order request.
+
+> ⚠ Take into account that this sample should only be used for demonstration purposes and does not reflect a fully production-ready implementation. We have chosen to only provide the bare bones of the application so the changes are clear.
```

Comment:
```suggestion
> ⚠ Take into account that this sample should only be used for demonstration purposes and does not reflect a fully production-ready implementation. We have chosen to only provide the bare bones of the application to be able to focus on the changes required for service-to-service correlation.
```

Reply:
Rather long for such a short purpose, but ok.

Reply:
https://github.com/arcus-azure/arcus.observability/pull/467/commits/1d7dc7d322057da07d44e7d48fb60b7cb478370e